web/client/product/assets/css/viewer.css

```diff
@@ -391,10 +391,7 @@ html, body, #container, .fill {
 
 #mapstore-catalog-panel {
     width: auto;
-    position: absolute;
-    right: 200px;
-    margin-top: 100px;
-    display: inline-block;
+    margin: 100PX;
```

Comment:
**not** capital px. The style should be responsive (if the width of the window is smaller, it should be 0, for instance).